### PR TITLE
Feature/res 83 release version 1.5 sample update

### PR DESCRIPTION
**Pull request by apichap-solutions**

### CODE REVIEW

File: petstore-api-mssql/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+|----------------------------------------|-----------|-------------|----------------|---------------------------------------------------------------------------------|
+| petstore-api-mssql                     | 5         | MSSQL       | -              | Petstore (Get, Save) with multiple SQL table relations in MSSQL                 |
+
+## Documentation
+Find the full documentation under: [docs.apichamp.com](https://docs.apichamp.com)
+
+## Requirements:
+- [Docker Client](https://docs.docker.com/get-started/overview/)
+
+> **You must accept the licence terms of MSSQL**: Accept the ACCEPT_EULA in your docker-compose.yaml file.
```
Comment (michael-weissenboeck) [inline petstore-api-mssql/README.md]: Is grey and easy to read over, but the info is sufficient to understand what needs to be done. The code part underneath is more prominent

**Review by michael-weissenboeck — APPROVED** [review]
for some examples, some background would be good. 
Petstore: what categories or tags are there?
Product: what categories can I filter for? (get without filter category returns list of products, so not that big of an issue)

**Review by michael-weissenboeck — APPROVED** [review]

File: petstore-api-mssql/README.md (hunk 1 above)
Comment (apichap-solutions) [reply petstore-api-mssql/README.md]: Added warning sign.

**Comment by apichap-solutions** [comment]
> for some examples, some background would be good. Petstore: what categories or tags are there? Product: what categories can I filter for? (get without filter category returns list of products, so not that big of an issue)

@michael-apichamp Added database tables for context and some more information for testing into openapi specs.

**Pull request merged**